zk-sdk/src/sigma_proofs/percentage_with_cap.rs

```diff
@@ -117,9 +97,9 @@ impl PercentageWithCapProof {
     /// * `transcript` - The transcript that does the bookkeeping for the Fiat-Shamir heuristic
     #[allow(clippy::too_many_arguments)]
     pub fn new(
-        percentage_commitment: &PedersenCommitment,
-        percentage_opening: &PedersenOpening,
-        percentage_amount: u64,
+        fee_commitment: &PedersenCommitment,
+        fee_opening: &PedersenOpening,
+        fee_amount: u64,
```

Comment:
Shouldn't this still be `percentage` to avoid language that's too specific to tokens?

Reply:
Yeah I forgot the original reason why we used `percentage`... I changed it back to `percentage`.